modules/math/src/geometry/iterators/primitive-iterator.ts

```diff
@@ -1,12 +1,24 @@
-/** @typedef {import('./primitive-iterator')} types */
 import {GL} from '../constants';
 import {getPrimitiveModeType} from '../primitives/modes';
 import {assert} from '@loaders.gl/loader-utils';
+import type {Information} from '../types';
```

Comment:
If the type is just used locally here and not exported from the module, just declare it locally. No need to move it into `types`.